### ISSUE TITLE
[Question] Is this going somewhere and how can I help?

### ISSUE THREAD

**Issue by julian-fairfax** [Levminer/authme-web#1]
Hi,
I've been on a quest to switch to as many open-sourcing privacy-respecting apps as possible. I'm happy to say that this process is mostly complete, but it has come with a trade-off for some apps, such as my authenticator app; the open source alternatives I'm now using aren't cross-platform.

This is why your app (/PWA) is so interesting to me. If this takes off then I'd finally have a fully open source, fully cross-platform authenticator app! I've yet to try your desktop app but since this app (/PWA) doesn't support actually adding or exporting any codes itself, I was just wondering whether you were planning on adding those features at some point? And if so, how can I help you do that? I've noticed that you're pretty much working on this project by yourself and so I'm ready to do whatever I can to support that development. Thanks so much for your work on this wonderful app and I hope it'll catch on!

**Comment by Levminer** [comment]
Hi!

Thank you for your interest! 

* Authme Web can support easily export your saved code, it's should be a copy/paste from the desktop app. 
* The import function is not that simple, if you are importing your own screen-shooted QR codes it can be implemented, but if you import from google authenticator it depends on python so it will not work on the web. 
* And if you want to support the projetct drop a star on this and the authme repo, the more stars the more GitHub advertise the project, or support at: https://paypal.me/levminer

So in conclusion, you have to use the desktop app to be able to use the web app. After that you can delete the desktop app if you want.

**Comment by julian-fairfax** [comment]
> The import function is not that simple, if you are importing your own screen-shooted QR codes it can be implemented, but if you import from google authenticator it depends on python so it will not work on the web.

Is it planned to uncouple the web app from the desktop at some point or is there some sort of technical limitation preventing that from being possible? I don't need the Google Authenticator stuff although a manual add option would work for that.

**Comment by Levminer** [comment]
Yes, the manual add can be implemented in the future, if you don't want to use the desktop app, but at this point a recommend using the desktop app and deleting it after.

**Comment by julian-fairfax** [comment]
Alright, I'm going to go ahead and close this issue but I'll be sure to check on this for future updates. Might replace my authenticator apps with it! Thanks for your help!

**Comment by Levminer** [comment]
Thank you for you interest!

**Comment by Levminer** [comment]
Hi @julian-fairfax 

As you requested I added the import function to the web app. It's on the dev site, you can try it out (https://authme-web.pages.dev) Let me know if its working

**Comment by julian-fairfax** [comment]
> Hi @julian-fairfax
> 
> As you requested I added the import function to the web app. It's on the dev site, you can try it out (https://authme-web.pages.dev) Let me know if its working for you.

It is working for me, yes

**Comment by Levminer** [comment]
Awesome! I have to fix some things, the changes going live tomorrow,
